meta/AccessTable.php

```diff
@@ -150,7 +152,7 @@ public function getDataColumn($column) {
      * @return Value[] a list of values saved for the current page
      */
     public function getData() {
-        $this->setCorrectTimestamp($this->pid, $this->ts);
+        $this->getLastRevisionTimestamp($this->pid, $this->ts);
```

Comment:
`getLastRevisionTimestamp()` now only returns a timestamp, without sideeffects. Shouldn't we use the retrieved value somehow?